ruby/lib/google/protobuf.rb

```diff
@@ -36,3 +39,25 @@
 end
 
 require 'google/protobuf/repeated_field'
+
+module Google
+  module Protobuf
+
+    def self.encode(msg)
+      msg.to_proto
+    end
+
+    def self.encode_json(msg)
+      msg.to_json
+    end
+
+    def self.decode(klass, proto)
+      klass.decode(proto)
+    end
+
+    def self.decode_json(klass, json)
+      klass.decode_json(json)
+    end
+
+  end
+end
```

Comment:
these methods existed in both jruby and mri, but I think it is cleaner to just define them once